conftest.py

```diff
@@ -6,11 +6,94 @@
 
 import six
 import pytest
+import sentry_sdk
+from sentry_sdk import start_transaction
+
 
 pytest_plugins = ["sentry.utils.pytest"]
 
 sys.path.insert(0, os.path.join(os.path.dirname(__file__), "src"))
 
+if os.environ.get("PYTEST_SENTRY_DSN"):
+    sentry_sdk.init(os.environ.get("PYTEST_SENTRY_DSN"), traces_sample_rate=1.0)
+
+
+txn = {}
+spans = {}
+call_spans = {}
```

Comment:
I see you use this to update spans. Do you have other reasons to store transactions and spans?

The current model of how transactions work, as soon as `finish` is called, the transaction is sent to Sentry. So this might be causing problems as you try to update and finish already finished transactions.